sources/packages/backend/libs/sims-db/src/entities/application-exception-requests.model.ts

```diff
@@ -71,4 +72,16 @@ export class ApplicationExceptionRequest extends RecordDataModel {
     nullable: true,
   })
   exceptionHash?: string;
+
+  /**
+   * Status of the application exception request.
+   */
+  @Column({
+    name: "exception_request_status",
+    type: "enum",
+    enum: ApplicationExceptionRequestStatus,
+    enumName: "ApplicationExceptionRequestStatus",
+    nullable: false,
```

Comment:
Please remove the `nullable: false` since it is the default value and we are no longer explicitly adding them.